sdk/resourcehealth/azure-resourcemanager-resourcehealth/src/test/java/com/azure/resourcemanager/resourcehealth/ResourceHealthTests.java

```diff
@@ -41,11 +44,21 @@ private static String randomPadding() {
     @Test
     @LiveOnly
     public void resourceHealthTest() {
+        TokenCredential credential = new AzurePowerShellCredentialBuilder().build();
+        AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        ResourceManager resourceManager = ResourceManager.configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
         ComputeManager computeManager = ComputeManager.configure()
+            .withPolicy(new ProviderRegistrationPolicy(resourceManager))
             .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
             .authenticate(new AzurePowerShellCredentialBuilder().build(), new AzureProfile(AzureEnvironment.AZURE));
```

Comment:
nit, use credential and profile defined above

Reply:
Fixed by [commit#7d90a3c](https://github.com/Azure/azure-sdk-for-java/pull/43440/commits/7d90a3ca278546258c5ce281dd8ffba04826f047)